### ISSUE TITLE
Editor bug: tab doesn't respond (macOS)

### ISSUE THREAD

**Issue by Lampogolovii** [defold/defold#6030]
Description
Often tab doesn't respond to clicks on it.
I can't tell how to reproduce it with 100% chance. I experience this bug every 20-30 minutes during my work.
I suspect that problem is only on macOS

How does it look: tab doesn't respond to clicks. I can't close it, can't select it, can't move it.
This bug reproduces on every Defold Editor version (on each 1.2.179 -- 1.2.186, for example)

Workaround
No matter what type of file opened in this tab. It could happen with any type - script, go, collection.
I deal with this bug by selecting this tab from "open assets" (command+P) window and then closing this tab by "close" (command+W). Then I can open this file again - and everything will work fine until this bug occurs again.  

I recorded a [video](https://www.youtube.com/watch?v=9vu9TQqJEEk) to represent how does it look
Description what's going on on the video:
0:00 - 0:10 Tab with keyboardController.script doesn't respond on any clicks. I can't select neither close it. Other tabs work fine
0:10 - 0:13 I can select this tab by "open assets" window
0:14 - 0:17 Ok, I can select it, but tab is still frozen - if I go to another tab I can't select my frozen tab again
0:18 - 0:27 Only after selecting by "open assets" and close by "close" menu - I can open this file again and deal with this bug

**Comment by britzl** [comment]
Duplicate of https://github.com/defold/defold/issues/5456

**Comment by AGulev** [comment]
@Lampogolovii are you able to reproduce it using the lates version of Defold?

**Comment by Lampogolovii** [comment]
@AGulev I've downloaded 1.2.189 (did you mean this version?)
I've been working with this version for awhile today. Yes, this bug is still reproducible. But less often!!

**Comment by AGulev** [comment]
I can't reproduce it lately (1.7.0, 1.8.0), can you?

**Comment by AGulev** [comment]
close as duplicate of https://github.com/defold/defold/issues/5456

**Comment by Lampogolovii** [comment]
I code in 1.7.0 and haven't seen this bug long time
